<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="v_t_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Velcoity Profile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="beam_on">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="beam_on_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>On (A)</string>
    </property>
   </widget>
   <widget class="QLabel" name="beam_t">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="beam_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Beam Profile</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="beam_off">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="beam_off_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Off (B)</string>
    </property>
   </widget>
   <widget class="QLabel" name="v_t_plot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Velocity_profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="beam_plot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Beam_profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="path_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Torch Path</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="path_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nx</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ny</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nz</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="path_addRow_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="path_removeRow_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>670</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dirx0">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="diry0">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dirz0">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dir0_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial direction vector:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dirx0_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>660</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>x0</string>
    </property>
   </widget>
   <widget class="QLabel" name="diry0_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>660</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>y0</string>
    </property>
   </widget>
   <widget class="QLabel" name="dirz0_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>660</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>z0</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="v_t_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Time (s)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Velocity (mm/s)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="v_t_removeRow_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="v_t_addRow_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="Line" name="split1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="split2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_apply">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>660</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="path_plot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>path_description.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
